oak/server/rust/oak_runtime/src/node/logger.rs

```diff
@@ -36,12 +38,29 @@ fn logger(pretty_name: &str, runtime: RuntimeRef, reader: ChannelReader) -> Resu
         let _ = runtime.wait_on_channels(&[Some(&reader)]);
 
         if let Some(message) = runtime.channel_read(&reader)? {
-            let message = String::from_utf8_lossy(&message.data);
-            info!("{} LOG: {}", pretty_name, message);
+            match LogMessage::decode(&*message.data) {
+                Ok(msg) => info!(
```

Comment:
I think eventually we should not be logging these log messages via the Rust logger, otherwise we end up with double log markers everywhere, and the levels don't quite make sense. Perhaps we should just use https://doc.rust-lang.org/std/macro.eprintln.html and have full control over the format.

Reply:
I agree! The current format is redundant/inconsistent. I am not sure if it is a good idea to get rid of the Rust logger. With the logger, the user code has the flexibility of using a specific logger implementation, and possibly log into file or other forms of storage. 
If logging to file or other storage is desirable perhaps we could change the logging format, using something like `env_logger::Builder` (not sure if it is allowed to use and mix more than one logger implementations). 
Or do we always log into `stdout`/`stderr`?